packages/expo-env-info/src/helpers.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
```

Comment:
```suggestion
import fs from 'fs/promises';
```
generally nicer to use this import so the `fs` usage is the same.